docs/examples/simple_mlj_demo/simple_mlj_demo.md

```diff
@@ -379,7 +368,7 @@ julia>
 Now lets do the tuning and see results:
 ```julia
 tuning = MLJTuning.TunedModel(
-    model=tuned_xgb(),
+    model=model_tpl,
```

Comment:
This particular change to the .md file follows the rest of [simple_mlj_example](https://github.com/IQVIA-ML/TreeParzen.jl/blob/master/docs/examples/simple_mlj_demo/simple_mlj_demo.md#construct-first-treeparzen-space) where `model_tpl` is used as this: `XGBoostRegressor = MLJ.@load XGBoostRegressor;
model_tpl = XGBoostRegressor()` but for the layered spaces it was needed to wrap this model in a container and then use that model container if the space was nested but this is no longer needed.

Reply:
I meant what the meaning of "tpl" is.

Reply:
I'm not quite sure if `tpl` has any particular meaning given that it's just an instance of a XGBoostRegressor.   Looking at the examples in this file showing parallelisation may be it's something to do with "task parallelisation"? But it can equally be just called a `model`.